index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/stylesheet.css" />
+    <title>Adopt-A-Dog</title>
+  </head>
+  <body>
+<!-- HEADER with background image, logo and tagline -->
+    <header>
+      <div class="header-image">
+        <img class="logo-img" src="images/adoptadog-logo.svg" alt="adopt a dog logo"/>
+        <p class="header-text">Seattle's premiere dog adoption agency</p>
+      </div>
+
+<!-- Checkout box section for header -->
+      <div class="checkoutbox">
+        <img class="cart-img" src="images/cart-icon.svg" alt="shopping cart icon" /> <span class="total-cost">$0.00</span>
+        <a class="checkout-button-top" href="checkout.html">Checkout</a>
+      </div>
+</header>
+
+
+<!-- NAVIGATION -->
+  <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+  </nav>
+<div class="format">
+  <div class="main-body">
+<div class="main-section">
```

Comment:
Watch your indenting, it's a lot easier to see what's nested in elements if they are indented right.